src/Servers/Connections.Abstractions/src/FileHandleType.cs

```diff
@@ -8,7 +8,18 @@ namespace Microsoft.AspNetCore.Connections;
 /// </summary>
 public enum FileHandleType
 {
+    /// <summary>
+    /// This API is unused and no longer supported.
```

Comment:
FYI @Tratcher  / @halter73. Doesn't appear to be used. David thought this might have been specific to Libuv.